### DISCUSSION TITLE
Can you do multivariate Time Series forecasting in pycaret?

### DISCUSSION THREAD

**Discussion by alex-christofalos** [pycaret/pycaret#3463] [Q&A]
I was wondering if the new PyCaret enables multivariate time series analysis and how can I enable it? 

When I put multivariate data into the time series model and assign a column it says "Univariate" still and there is nothing in the documentation that shows how to change it to multivariate. 

Thanks in advance!!

**Comment by ngupta23** [comment]
pycaret only supports univariate analysis at this time.
